tests/utils/shopper.js

```diff
@@ -354,5 +354,11 @@ export const shopper = {
 
 			await expect( page.$x( cartItemXPath ) ).resolves.toHaveLength( 1 );
 		},
+
+		isLoggedIn: async () => {
```

Comment:
This checks if user is logged in as an admin, so maybe we could change this to `isLoggedInAdmin`?

I would also move this function directly on the `shopper` object as it's not directly related to blocks

Reply:
Hi @alexflorisca! Thank you for the review. And thank you for pointing this out. My intention wasn't to check if a user is logged in as admin only, but other types of accounts (as a customer). I changed the implementation to do the correct check and applied the suggestions. Please could you have a second look at it! 🙌